src/k8s/pkg/component/component_test.go

```diff
@@ -61,6 +61,12 @@ func buildChart(opts ...chartOption) *chart.Chart {
 	return c.Chart
 }
 
+func withName(name string) chartOption {
```

Comment:
Why don't we pass the chart name straight into the buildChart function?
You should be able to do something like this:
```
myChart := buildChart(
    func(opts *chartOptions) {
        opts.Chart.Metadata.Name = "custom-chart"
    },
   ```
   The opts for-loop in `buildChart` should be able to set the chart options for you.

Reply:
In upstream code there is withName, withNamespace, etc.
https://github.com/helm/helm/blob/15d116b8548f59959fe9f2fafdaf7c6de8db3ea0/pkg/action/action_test.go#L135